### PR TITLE
[Frontend] - Search Bar Component

### PR DESCRIPTION
**Pull request by sandrex-SGV**
## Ticket/s

- [Search Bar Component](https://trello.com/c/D9UbQPCC/23-search-bar-component)

## Changes

- Create Searchbar component 

#### Figma
image

#### Actual
image

image

## How to test?

- Run `pnpm run dev` script
- Go to http://localhost:5173

### CODE REVIEW

File: apps/web/src/views/SampleView.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { Dropdown } from '../../../../packages/ui/dropdown/dropdown';
+import { Searchbar } from '../../../../packages/ui/searchbar/searchbar';
 export const SampleView = () => {
   return (
     <>
-      <Dropdown />
+      {/* <Dropdown /> */}
```
Comment (dandalandone) [inline apps/web/src/views/SampleView.tsx]: remove if not used

File: packages/ui/searchbar/searchbar.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+export const Searchbar = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        height: '70px',
```
Comment (dandalandone) [inline packages/ui/searchbar/searchbar.tsx]: move all styling to styled Components, ask @brentmillen how to move.

**Review by markjoshuacarmen — APPROVED** [review]
looks nice to me!

**Review by TreshAnn — APPROVED** [review]
LGTM!

File: packages/ui/searchbar/searchbar.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            radius="xl"
+            style={{
+              marginRight: '5px',
+              width: '100px',
+              borderBottom: '1px solid #00000040',
+              borderRadius: '20px',
+              boxShadow: '0px 2px 2px 0px rgba(0, 0, 0, 0.25)',
+            }}
+            data={[
+              { value: 'latest', label: 'Latest' },
```
Comment (dandalandone) [inline packages/ui/searchbar/searchbar.tsx]: Move mock data to sample file

**Review by dandalandone — DISMISSED** [review]

**Review by brentmillen — APPROVED** [review]
lgtm!

**Comment by SGVGen** [comment]
LGTM!

**Review by DanDalanDone69 — APPROVED** [review]
LGTM!

**Pull request merged**